contracts/interfaces/UserOperation.sol

```diff
@@ -7,13 +7,12 @@ pragma solidity ^0.8.12;
  * @param nonce                 - Unique value the sender uses to verify it is not a replay.
  * @param initCode              - If set, the account contract will be created by this constructor/
  * @param callData              - The method call to execute on this account.
- * @param callGasLimit          - The gas limit passed to the callData method call.
- * @param verificationGasLimit  - Gas used for validateUserOp and validatePaymasterUserOp.
+ * @param accountGasLimits      - Packed gas limits for validateUserOp and gas limit passed to the callData method call.
  * @param preVerificationGas    - Gas not calculated by the handleOps method, but added to the gas paid.
  *                                Covers batch overhead.
  * @param maxFeePerGas          - Same as EIP-1559 gas parameter.
  * @param maxPriorityFeePerGas  - Same as EIP-1559 gas parameter.
- * @param paymasterAndData      - If set, this field holds the paymaster address and paymaster-specific data.
+ * @param paymasterAndData      - If set, this field holds the paymaster address, verification gas limit, postOp gas limit and paymaster-specific extra data
  *                                The paymaster will pay for the transaction instead of the sender.
  * @param signature             - Sender-verified signature over the entire request, the EntryPoint address and the chain ID.
  */
```

Comment:
we said the on-chain type should be called "PackedUserOp", didn't we?

Reply:
Fixed